src/priyanka/Assignment_18/MaxRepeatingWord.java

```diff
@@ -0,0 +1,38 @@
+package priyanka.Assignment_18;
+
+/*Program : 1
+max repeating word with its freq.
+input : Hi Hello Hi Techno Hello Hi
+output : Hi -> 3*/
+
+public class MaxRepeatingWord {
+
+	void maxRepeatWord(String str) {
+
+		String[] strArr = str.split(" ");
+		int max = 0;
+		String maxWord = "";
+		for (int index = 0; index < strArr.length; index++) {
+			int count = 0;
+			String word = strArr[index];
+			if (str.contains(word)) {
```

Comment:
Here you are putting one word from array to the variable called "word", & then you are checking that variable value into array, so here think logically, it will always gives you true right?
storing in one variable and checking same variable in that array makes no sense here.